README.md

```diff
@@ -1 +1,50 @@
-# anycloud
+# AnyCloud
+
+AnyCloud is an Lambda alternative for Node.js that works with any cloud provider. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
+
+- Vendor lock-in
+- Cold starts
+- Limited runtime (10-15m)
+- Stateless
+- Cumbersome to run locally
+- HTTPS support not included
+
+## Documentation
+
+Visit our [docs](https://alantech.gitbook.io/anycloud)
+
+## Status
+- [ ] Public Alpha: Anyone can sign up, but go easy on us
+- [ ] Public Beta: Stable enough for most non-enterprise use-cases
+- [ ] Public: Production-ready
+
+## How it works
+
+AnyCloud is a serverless framework built on the Alan and Rust programming languages that:
+
+- [x] Automatically scales your http server across many machines based on request load and system stats
+- [x] Works on multiple cloud providers so you are not locked into AWS
+- [x] Runs locally as-is without special configuration of your local dev environment
+- [ ] In-memory distributed datastore
```

Comment:
Should we include this right now, especially since we haven't even documented datastore in the Alan docs?

Reply:
yes, it is not checked off so it seems fine by me